config/route_helper.cr

```diff
@@ -1,5 +1,5 @@
-Lucky::RouteHelper.configure do
-  if Lucky::Env.production?
+Lucky::RouteHelper.configure do |settings|
+  if ENV["APP_DOMAIN"]?
```

Comment:
Aahh that was still a local change, had to make that to generate proper sitemap I think.
```suggestion
  if Lucky::Env.production?
```